lib/v2/eastmoney/search/index.js

```diff
@@ -0,0 +1,46 @@
+const cheerio = require('cheerio');
+const { parseDate } = require('@/utils/parse-date');
+
+const PuppeterGetter = async (ctx, browser, link) => {
+    const result = await ctx.cache.tryGet(`eastmoney: ${link}`, async () => {
+        const page = await browser.newPage();
+        await page.goto(link);
+        await page.waitForSelector('.news_list');
+        const html = await page.evaluate(() => document.documentElement.innerHTML);
+        return html;
+    });
+    return result;
+};
```

Comment:
No need to use puppeteer at all. You can send request to `search-api-web.eastmoney.com`

Reply:
There's no API Reference for that. Simply look at the network tab in DevTools when visiting `https://so.eastmoney.com/News/s?KeyWord=${keyword}`

Reply:
Of course I have searched the host in Devtools before. I cannot find any request with 'search' like host.

Reply:
There is just one xhr request. But the response have nothing about search result.

Reply:
They use jQuery callback instead of XHR.
![image](https://user-images.githubusercontent.com/11386903/232076699-4b371825-8060-4b4f-9cde-23ff636203c6.png)

Reply:
Thankyou so much! I found it and fixed.